bench/Directory.Build.props

```diff
@@ -15,6 +15,6 @@
 
     <SignAssembly Condition="'$(OS)' == 'Windows_NT'">true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\RavenDB.snk</AssemblyOriginatorKeyFile>
-
+	<AssemblyOriginatorKeyFile Condition="EXISTS('.\..\..\..\..\..\..\RavenDB.snk')">.\..\..\..\..\..\..\RavenDB.snk</AssemblyOriginatorKeyFile>
```

Comment:
Maybe it would be good to apply that fix to 4.2? It will be merged to never versions eventually

Reply:
BenchmarkDotNet library generates .csproj under the covers. Fix comes from: https://github.com/dotnet/BenchmarkDotNet/issues/836